sdk/daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/UpgradesSpec.scala

```diff
@@ -598,12 +565,9 @@ abstract class UpgradesSpec(val suffix: String)
     bytes.map((dar.main.pkgId, _))
   }
 
-  def uploadPackagePair(
-      path: Upgrading[String]
-  ): Future[Upgrading[(PackageId, Option[Throwable])]]
-
```

Comment:
What was this for previously, and why isn't it needed now?

Reply:
I defined `uploadPackage`, and now we reuse that directly in `testPackagePair` and `testPackageTriple`, so we no longer need the dedicated "upload both" function. It allows more granularity on how we upload each package in different tests.